cpp/util_module/util_module.cpp

```diff
@@ -5,10 +5,12 @@
 extern "C" int mgp_init_module(struct mgp_module *module, struct mgp_memory *memory) {
   try {
     mgp::MemoryDispatcherGuard guard{memory};
-    AddProcedure(Util::Md5, std::string(Util::kProcedureMd5).c_str(), mgp::ProcedureType::Write,
-                 {mgp::Parameter(std::string(Util::kArgumentValuesMd5).c_str(), {mgp::Type::List, mgp::Type::Any})},
+    AddProcedure(Util::Md5Procedure, std::string(Util::kProcedureMd5).c_str(), mgp::ProcedureType::Write,
```

Comment:
Should be a read procedure since it doesn’t modify the graph:
```suggestion
    AddProcedure(Util::Md5Procedure, std::string(Util::kProcedureMd5).c_str(), mgp::ProcedureType::Read,
```